src/utils/cleanupLotion.js

```diff
@@ -30,12 +31,21 @@ module.exports = async app => {
       )
     );
 
-    await rimraf(path.join(lotionPath, 'data'));
+    await rimraf(dataPath);
     await rimraf(path.join(lotionPath, 'merk'));
     await rimraf(path.join(lotionPath, 'leap.db'));
     await rimraf(path.join(configPath, 'addrbook.json'));
     await rimraf(path.join(configPath, 'config.toml'));
     await rimraf(path.join(configPath, 'genesis.json'));
+    fs.mkdirSync(dataPath);
+    await writeFile(
```

Comment:
Doesn't tendermint takes care of that? 🤔

Reply:
with `--fresh` we want to drop everything by the key. If we leave only `priv_validator_key.json`, tendermint will complain about missing state file.

It wasn't an issue before because of different priv_validator format (no separate state file)